tests/verify_with_chdir.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Steven Bosnick
```

Comment:
This was split off into it's own test dir to prevent a chdir race condition impacting the other verify tests.